src/components/Table/Table/__stories__/helpers.js

```diff
@@ -259,5 +259,6 @@ export const TableAvatar = ({ text }) => (
     size={Avatar.sizes.SMALL}
     ariaLabel={text}
     backgroundColor="dark_purple"
+    tabIndex={-1}
```

Comment:
This smells bad.
The fact that we're not using the Avatar as is and we need to trick it with tabIndex means every consumer will need to do it.

Also, the fact that it's a helper component means it won't appear in the code example as copy-pastable

Reply:
I agree, it's currently for the purpose of future-story in storybook, to not focus on it.
currently, we don't have story for the highlight, Yael wants to come up with one and then I'll add it.

As we apply tabIndex 0 to avatar by default, it causes this headache and it would cause others the headache as well (and I guess even today it is, I don't see a reason to focus on an Avatar if it has no onClick event).
But removing this tabIndex from avatar would cause breaking change, so I'm pretty much blocked here :/

Reply:
We recently removed tabIndex = 0 from Avatar. I didn't feel like it's super breaking change actually, so we did it as a part of Hacktoberfest - https://github.com/mondaycom/monday-ui-react-core/pull/1594
Instead we introduced logic which makes tabIndex = 0, if there is `onClick` or tooltip content is not empty
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/1b4328ca-35f6-4fb1-9db5-74eef495f05b)

It doesn't solve this issue though! Cause tooltip content isn't empty, so the Avatar is focusable by default

Reply:
Done ✅